src/warnings.cpp

```diff
@@ -47,8 +47,11 @@ std::string GetWarnings(const std::string& strFor)
     LOCK(cs_warnings);
 
     if (!CLIENT_VERSION_IS_RELEASE) {
-        strStatusBar = "This is a pre-release test build - use at your own risk - do not use for mining or merchant applications";
-        strGUI = _("This is a pre-release test build - use at your own risk - do not use for mining or merchant applications");
+        const char *PRE_RELEASE_MSG =
+            "This is a pre-release test build - use at your own risk - do not use for "
+            "mining or merchant applications.";
+        strStatusBar = PRE_RELEASE_MSG;
+        strGUI = _(PRE_RELEASE_MSG);
```

Comment:
Not sure if the changes are worth it. Is this even picked up as a translation that way?

Reply:
Is your concern with the adjacent string literals? That should not affect translation. Or am I missing something?